firelink/apps.py

```diff
@@ -51,6 +51,43 @@ def _process_bonfire_apps_list(self, apps):
         apps_array.sort(key=lambda app: app['name'])
         return apps_array
 
+    def get_processed_template(self, request):
+        """Get the processed template for an app."""
+        namespace = request["namespace"]
+        clowd_env = self._get_clowdenv_for_ns(namespace)
+        if not clowd_env:
+            raise bonfire.FatalError("Could not find a ClowdEnvironment tied to ns '{ns}'.")
+        request["clowd_env"] = clowd_env
+        apps_config = bonfire._process(
+            request["app_names"],
+            request["source"],
+            request["get_dependencies"],
+            request["optional_deps_method"],
+            request["local_config_method"],
+            request["set_image_tag"],
+            request["ref_env"],
+            request["fallback_ref_env"],
+            request["target_env"],
+            request["set_template_ref"],
+            request["set_parameter"],
+            request["clowd_env"],
+            request["local_config_path"],
+            AppOrComponentSelector(bool(request.get("remove_resources")), request.get("remove_resources", []), []),
```

Comment:
This would set `select_all` to `True` if the `remove_resources` key is present/populated in `request`.

I think the only time `select_all` should be True is if `all` is in `remove_resources` ... or if the "all" box is checked on the UI

Reply:
I sent up another revision that I hope does the right thing this time.